repomgr/repomgr.go

```diff
@@ -957,33 +967,7 @@ func (rm *RepoManager) processNewRepo(ctx context.Context, user util.Uid, r io.R
 	}
 
 	// mild hack: without access to the 'meta' object, we cant properly verify each new repo slice has the right DID in the case of a gap fill procedure
-	if until.Defined() {
-		robs, err := rm.cs.ReadOnlySession(user)
-		if err != nil {
-			return err
-		}
-
-		old, err := repo.OpenRepo(ctx, robs, until, true)
-		if err != nil {
-			return err
-		}
-
-		mcid, err := old.MetaCid(ctx)
-		if err != nil {
-			return err
-		}
-
-		blk, err := robs.Get(ctx, mcid)
-		if err != nil {
-			return err
-		}
-
-		if err := membs.Put(ctx, blk); err != nil {
-			return err
-		}
-
-		membs = util.NewReadThroughBstore(robs, membs)
-	}
+	// TODO(bnewbold): @Why should review the code that was removed here
```

Comment:
This is good, this was the hack necessary for the meta object being a separate thing. I'm only sad that i wasnt the one to delete it ;)

Reply:
Ok, I'll pull this comment out.